app.js

```diff
@@ -37,7 +37,11 @@ var auth = require('./app/auth/index'),
 if (httpsEnabled) {
      app = express().https({
         key: fs.readFileSync(settings.https.key),
-        cert: fs.readFileSync(settings.https.cert)
+        cert: fs.readFileSync(settings.https.cert),
+        honorCipherOrder: settings.https.honorCipherOrder || true,
```

Comment:
Seems reasonable. Does this block old clients or some such thing? When would you ever want to allow the client to dictate what to use?